errbot/backends/irc.py

```diff
@@ -416,6 +430,21 @@ def on_dcc_connect(self, dcc, event):
     def on_dcc_disconnect(self, dcc, event):
         self.transfers.pop(dcc)
 
+    def on_endofnames(self, c, e):
```

Comment:
You seem to have a pretty clear idea of what the `endofnames` represents and when it is fired. Could you add a docstring to the function briefly explaining it? That way it's clear to everyone (I wouldn't know without looking it up right now, for example).